api-admin/src/main/java/bio/terra/pearl/api/admin/controller/forms/ConfiguredSurveyController.java

```diff
@@ -61,4 +61,15 @@ public ResponseEntity<Object> create(
             portalShortcode, studyShortcode, environmentName, configuredSurvey, adminUser);
     return ResponseEntity.ok(savedSes);
   }
+
+  @Override
+  public ResponseEntity<Void> remove(
+      String portalShortcode, String studyShortcode, String envName, UUID configuredSurveyId) {
+    AdminUser adminUser = requestService.requireAdminUser(request);
+    EnvironmentName environmentName = EnvironmentName.valueOfCaseInsensitive(envName);
+
+    surveyExtService.removeConfiguredSurvey(
+        portalShortcode, studyShortcode, environmentName, configuredSurveyId, adminUser);
+    return ResponseEntity.noContent().build();
+  }
```

Comment:
I'm still in the process of adding backend test coverage but that shouldn't block getting started on reviews, imo